scipy/stats/tests/test_stats.py

```diff
@@ -73,22 +73,21 @@
 ROUND = array([0.5,1.5,2.5,3.5,4.5,5.5,6.5,7.5,8.5], float)
 
 
+@array_api_compatible
+@skip_xp_backends('array_api_strict', 'jax.numpy',
+                  reasons=["`array_api_strict.where` `fillvalue` doesn't "
+                           "accept Python floats. See data-apis/array-api#807.",
+                           "JAX doesn't allow item assignment."])
+@pytest.mark.usefixtures("skip_xp_backends")
 class TestTrimmedStats:
     # TODO: write these tests to handle missing values properly
     dprec = np.finfo(np.float64).precision
 
-    @array_api_compatible
-    @skip_xp_backends('array_api_strict', 'jax.numpy',
-                      reasons=["`array_api_strict.where` `fillvalue` doesn't "
-                               "accept Python floats. See data-apis/array-api#807.",
-                               "JAX doesn't allow item assignment, and this  "
-                               "function uses _lazywhere."])
-    @pytest.mark.usefixtures("skip_xp_backends")
     def test_tmean(self, xp):
-        x = xp.asarray(X)
+        x = xp.asarray(X.tolist())  # use default dtype of xp
```

Comment:
This is a neat trick I hadn't thought of doing this

Reply:
How portable is this out of interest?

Reply:
`tolist` says it converts NumPy scalars to Python scalars, and it sounds like a Python `float` is [all but guaranteed](https://stackoverflow.com/questions/70184494/on-what-systems-does-python-not-use-ieee-754-double-precision-floats) to be an IEEE 754 double.